bears/vcs/git/GitConflictBear.py

```diff
@@ -0,0 +1,88 @@
+import shutil
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.results.Diff import Diff
+
+# Run test at coala-bears/tests/vcs/git with
+# coala --bears=GitConflictBear --files=
+# conflict.txt,noconflict.txt
+# -L DEBUG --flush-cache
+
+
+class GitConflictBear(LocalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    @classmethod
+    def check_prerequisites(cls):
+        if shutil.which('git') is None:
+            return 'git is not installed.'
+        else:
+            return True
+
+    def run(self, filename, file):
+        # Possible parameters : from line, to line
```

Comment:
We can't have a comment here. We use automatic parsing for documentation (from inline docs) and I think it uses the first comment as the description.